assets/css/v2/eink/departures_no_data.scss

```diff
@@ -0,0 +1,58 @@
+.departures-no-data-container {
+  width: 1200px;
+  height: 1312px;
+}
+
+.departures-no-data__main-content {
+  margin-bottom: 53px;
+  margin-left: 208px;
+}
+
+.departures-no-data__main-content__no-connection-icon-container {
+  width: 236px;
+  height: 236px;
+  margin-bottom: 48px;
+  margin-top: 111px;
+}
+
+.departures-no-data__main-content__heading {
+  width: 770px;
+  height: 448px;
+  color: rgb(0, 0, 0);
+  font-size: 104px;
+  font-family: Inter, sans-serif;
+  font-weight: bold;
+  letter-spacing: 0px;
+  line-height: 112px;
+}
+
+.departures-no-data__hairline {
+  content: "";
+  background: #171f26;
```

Comment:
Hmm, I thought for e-ink we only used black/white/grayscale. Could you check with Mindy that this is definitely the color we want?

Reply:
I'll double-check with design before merging. E-Ink NoData state also uses this same color for the divider, so I'm thinking it's ok. But they might also both be incorrect.

Reply:
Great callout on this. Checked with Mindy and she agreed that these should be solid black. Went ahead and did this for `NoData` as well.